.envrc

```diff
@@ -5,6 +5,16 @@
 # If you'd like to override or set any custom environment variables, this .envrc will read a .env file at the end.
 set -e
 
+# Upgrading Mac can uninstall the Command Line Tools, thus, removing our access to git
+# The message talks about xcrun, however, we can use the lack of git as a way to know that we need this
+# xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools),
+# missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
```

Comment:
Would it be better to check for `xcrun`?

Reply:
The output is odd. `xcrun` never gets installed there.
```
(.venv) armenzg@Armens-M1 sentry % ls -l /usr/bin | grep xcrun
-rwxr-xr-x  1 root   wheel    137152 17 Oct 23:30 xcrun
(.venv) armenzg@Armens-M1 sentry % ls -l /Library/Developer/CommandLineTools/usr/bin | grep xcrun
```

Reply:
Oh weird